sirepo/db_upgrade.py

```diff
@@ -85,6 +85,22 @@ def _20240507_cloudmc_to_openmc(qcall):
             _migrate_sim_type("cloudmc", "openmc", qcall, u)
 
 
+def _20240524_add_role_user(qcall):
+    if not qcall.auth_db.table_exists("user_role_invite_t"):
+        return
+    qcall.auth_db.execute_sql(
+        """
+INSERT INTO user_role_moderation_t (uid, role, status, moderator_uid, last_updated)
```

Comment:
Ident this so easier to read. SQL doesn't care about leading whitespace.

Reply:
[793c0b5](https://github.com/radiasoft/sirepo/pull/7070/commits/793c0b547ac000e96d88c0a70af24f34c7ad011f)